api/base/serializers.py

```diff
@@ -269,8 +288,12 @@ def __init__(self, kind=None, kwargs=None, **kws):
 
     def get_url(self, obj, view_name, request, format):
         if self.kind and obj.kind != self.kind:
-            return None
-        return reverse(view_name, kwargs={attr_name: getattr(obj, attr) for (attr_name, attr) in self.kwargs}, request=request, format=format)
+            raise SkipField
+        url = reverse(view_name, kwargs={attr_name: getattr(obj, attr) for (attr_name, attr) in self.kwargs}, request=request, format=format)
+        if not url:
```

Comment:
See above comment; may not be necessary if `reverse` never returns a falsy value.